Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -57,7 +57,7 @@ NATIVEDIR=$VFS_SRCDIR/GVFS/GVFS.Native.Mac
 xcodebuild -configuration $CONFIGURATION -workspace $NATIVEDIR/GVFS.Native.Mac.xcworkspace build -scheme GVFS.Native.Mac -derivedDataPath $VFS_OUTPUTDIR/GVFS.Native.Mac || exit 1
 
 USERNOTIFICATIONDIR=$VFS_SRCDIR/GVFS/GVFS.Notifications/VFSForGit.Mac
-xcodebuild -configuration $CONFIGURATION -project $USERNOTIFICATIONDIR/VFSForGit.xcodeproj build -scheme "VFS For Git" -derivedDataPath $VFS_OUTPUTDIR/GVFS.Notifications/VFSForGit.Mac || exit 1
+xcodebuild -configuration $CONFIGURATION -project $USERNOTIFICATIONDIR/VFSForGit.xcodeproj build test -scheme "VFS For Git" -derivedDataPath $VFS_OUTPUTDIR/GVFS.Notifications/VFSForGit.Mac || exit 1
```

Comment:
@pmj @jeschu1 this change reminded me that I saw that `xcodebuild` has a `build-for-testing` action as well:

```
build                  Build the target in the build root (SYMROOT).  This is the default action, and is used if no action is given.

build-for-testing      Build the target and associated tests in the build root (SYMROOT).  This will also produce an xctestrun file in the build root. This requires specifying a scheme.
```

Do you know if we ever *have* to use `build-for-testing`?  Or will `test` ensure that whatever else needs to be built is built?

Reply:
`xcodebuild test` builds all its dependencies and runs the UT. `xcodebuild build-for-testing` is useful for distributed testing, which we don't do. So we don't need it now. 

More info - `build-for-testing` creates a `xctestrun` file. This file can be specified in the Xcode action `test-without-building` to speed up distributed testing - build once and run tests across a distributed set of machines (without needing to build on each one.) A quick search [result](https://medium.com/xcblog/speed-up-ios-ci-using-test-without-building-xctestrun-and-fastlane-a982b0060676).